new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItCoherenceTests.java

```diff
@@ -0,0 +1,394 @@
+
+  // This method is needed in this test class, since the cleanup util
+  // won't cleanup the images.
+  @AfterAll
+  void tearDown() {
+    // Delete domain custom resource
+    logger.info("Delete domain custom resource in namespace {0}", domainNamespace);
+    assertDoesNotThrow(() -> deleteDomainCustomResource(domainUid, domainNamespace),
+        "deleteDomainCustomResource failed with ApiException");
+    logger.info("Deleted Domain Custom Resource " + domainUid + " from " + domainNamespace);
+  }
+
+  /**
+   * Create a WebLogic domain with a Coherence cluster and deploying it using WDT
+   * Test rolling restart of Coherence managed servers and verify
+   * that data are not lost during a domain restart.
+   */
+  @Test
+  @DisplayName("Create domain with a Coherence cluster using WDT and test rolling restart")
+  @Slow
+  @MustNotRunInParallel
+  public void testRollingRestart() {
+    final String successMarker = "CACHE-SUCCESS";
+
+    // create and verify WebLogic domain image using model in image with model files
+    String miiImage = createAndVerifyDomainImage();
+
+    // create and verify a WebLogic domain with a Coherence cluster
+    createAndVerifyDomain(miiImage);
+
+    // build the Coherence proxy client program in the server pods
+    // which load and verify the cache
+    copyCohProxyClientAppToPods();
+
+    // run the Coherence proxy client to load the cache
+    String serverName = managedServerPrefix + "1";
+    final ExecResult execResult1 = assertDoesNotThrow(
+        () -> runCoherenceProxyClient(serverName, OP_CACHE_LOAD),
+        String.format("Failed to call Coherence proxy client in pod %s, namespace %s",
+            serverName, domainNamespace));
+
+    assertAll("Check that the cache loaded successfully",
+        () -> assertTrue(execResult1.exitValue() == 0, "Failed to load the cache"),
+        () -> assertTrue(execResult1.stdout().contains(successMarker), "Failed to load the cache")
+    );
+
+    logger.info("\n Coherence proxy client {0} returns {1} \n ",
+        OP_CACHE_LOAD, execResult1.stdout());
+
+    // patch domain to rolling restart it by change restartVersion
+    rollingRestartDomainAndVerify();
+
+    // build the Coherence proxy client program in the server pods
+    // which load and verify the cache
+    copyCohProxyClientAppToPods();
+
+    // run the Coherence proxy client to verify the cache contents
+    final ExecResult execResult2 = assertDoesNotThrow(
+        () -> runCoherenceProxyClient(serverName, OP_CACHE_VALIDATE),
+        String.format("Failed to call Coherence proxy client in pod %s, namespace %s",
+            serverName, domainNamespace));
+
+    assertAll("Check that the cache loaded successfully",
+        () -> assertTrue(execResult1.exitValue() == 0, "Failed to validate the cache"),
+        () -> assertTrue(execResult2.stdout().contains(successMarker), "Failed to validate the cache")
+    );
+
+    logger.info("\n Coherence proxy client {0} returns {1} \n ",
```

Comment:
Do you need a new line in each log message.  Same comment on previous log statements.

Reply:
newline signs removed